docs/GETTING_STARTED_GUIDE.md

```diff
@@ -22,6 +22,23 @@ First and foremost, the metrics service requires a Kubernetes cluster that align
 | --------- |
 | 1.17-1.19 |
 
+## Supported Dell EMC Products
+
+This project currently supports the following Dell EMC storage systems.
```

Comment:
"This project currently supports the following versions of Dell EMC PowerFlex."

Just list the supported versions in the table please